rudder-server-root/SPECS/rudder-server-root.spec

```diff
@@ -52,7 +52,7 @@ Source8: rudder-server-root
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 BuildArch: noarch
 
-Requires: rudder-jetty, rudder-webapp, rudder-inventory-endpoint, rudder-inventory-ldap, rudder-reports, rudder-agent, curl
+Requires: rudder-jetty, rudder-webapp, rudder-inventory-endpoint, rudder-inventory-ldap, rudder-reports, Rudder::Agent, curl
```

Comment:
Well, something has to be done, this war zypper willing not update rudder-agent on sles... I have to repackage basically all rudder-agent packages by http://rpmrebuild.sourceforge.net